protocol/0091-LIQM-liquidity_mechanisms.md

```diff
@@ -0,0 +1,118 @@
+
+Updates to this parameter will be used the next time LPs are designated, i.e. updating the network parameter will not result in LPs instantly being redesignated.
+
+## Accruing ELS Points
+
+Whenever a market proposal is enacted (passes opening auction), an internal Vega asset is created to track the ELS points of that market. For a successor market, a new asset should not be created and instead the ELS asset of the parent market should be used.
+
+At the end of each epoch, each party accrues ELS points for the relevant market as follows:
+
+$$ELS_{i_j} = V_{i_j} \cdot 10^{-Q}$$
+
+Where:
+
+- $ELS_{j}$ is the ELS points accrued by party $j$ in epoch $i$
+- $V_{i_j}$ is the notional maker fees of party $j$ in epoch $i$
+- $Q$ is the quantum of the asset in which the markets prices are expressed in (settlement for future and perpetual markets, quote for spot markets) 
+
+> [!NOTE]
+> the maker fees is scaled by the assets quantum such that 1 ELS point is earned for approx. every 1 USD of maker fees received.
+
+## Designating Liquidity Providers
+
+A party will only be able to receive liquidity fees or earn liquidity rewards providing they are designated as an LP for that epoch.
+
+A party will only be designated as an LP providing they received more than a specified proportion of the markets total maker fees in the previous epoch, let this requirement be $N$ (the network parameter `liquidityProvider.proportionRequirement`). Throughout the epoch the network will track each parties maker fees, $M$.  At the end of epoch $i$, a party will be designated as an LP for epoch $i+1$ providing:
+
+$$\frac{M_{i_j}}{\sum_{k}^{n}{M_{i_j}}} >= N$$
+
+Where:
+
+- $M_{i_j}$ is the maker fees of party ${j}$ in epoch ${i}$
+- $N$ the requirement specified by the network parameter `liquidityProvider.proportionRequirement` 
+
+By designating parties as LPs on an epoch by epoch basis the protocol ensures:
+
+- "in-active" parties with a large number of ELS points will no longer be rewarded through liquidity mechanisms should they stop providing liquidity (or provide uncompetitive liquidity).
+- "late-arriving" parties with a small number of ELS points will be rewarded through liquidity mechanisms if they provide competitive liquidity and as such comprise a larger proportion of the markets volume.
+
+
+## Implied Commitment Amount
+
+Each LPs `implied commitment amount` is defined as, the maximum volume of notional liquidity that they supplied within a specified range for at least N % of the epoch (where the liquidity range and N are market configurable parameters).
+
+To calculate the implied commitment amount, throughout the epoch, the network must sample and store the volume of notional liquidity supplied by each LP at that point. For now this is done once a block but could be sampled randomly to reduce the amount of data stored.
```

Comment:
"implied commitment" -> "volume of notional" 

Consider whether we need weighting.